sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m

```diff
@@ -337,8 +337,8 @@ - (void)bridgeCurrentItemChangedNotification:(NSNotification *)notification {
   
   //OS: Flag must be changed depends on if current asset has new is-live-status
   self.liveHelper = (struct LiveAssetHelper) {self.adjustedPlayhead, self.player.currentItem.live};
-  
-  [self playForJustChangedItem];
+  #warning: Required for cases of using old API [self.player setEmbedCode:embedCode]. Remove when SDK version > 4.46.0_GA. Also method '-playForJustChangedItem'  itself
```

Comment:
Also method itself what? :)

Reply:
Changed. I mean we should remove commented call in `- (void)bridgeCurrentItemChangedNotification:(NSNotification *)notification` and declaration of method `-playForJustChangedItem`